web/src/ResourceStatusSummary.tsx

```diff
@@ -109,29 +108,24 @@ type ResourceGroupStatusItemProps = {
 export function ResourceGroupStatusItem(props: ResourceGroupStatusItemProps) {
   const count = (
     <>
-      <ResourceGroupStatusSummaryItemCount>
+      <ResourceGroupStatusSummaryItemCount aria-label={`${props.label} count`}>
         {props.count}
       </ResourceGroupStatusSummaryItemCount>
       {props.countOutOf && (
         <>
           /
-          <ResourceGroupStatusSummaryItemCount>
+          <ResourceGroupStatusSummaryItemCount aria-label="Out of total resource count">
             {props.countOutOf}
           </ResourceGroupStatusSummaryItemCount>
         </>
       )}
     </>
   )
 
-  const descriptiveCount = (
-    <>
-      {count} <SrOnly>&nbsp; {props.label}</SrOnly>
```

Comment:
i changed the a11y markup here from additional inline text to using aria labels, since it's easier to test. i'm not sure if there's a meaningful difference between the two approaches for screen readers and other assistive tech.